etcdctlv3/main.go

```diff
@@ -46,6 +47,8 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&globalFlags.OutputFormat, "write-out", "w", "simple", "set the output format (simple, json, protobuf)")
 	rootCmd.PersistentFlags().BoolVar(&globalFlags.IsHex, "hex", false, "print byte strings as hex encoded strings")
 
+	rootCmd.PersistentFlags().DurationVar(&globalFlags.DialTimeout, "dial-timeout", 5*time.Second, "dial timeout for client connections")
```

Comment:
can we change the default timeout to 2 second? or less? also can we make the default dial timeout a const?